nixos/doc/manual/development/option-types.xml

```diff
@@ -23,30 +23,74 @@
         <literal>false</literal>.</para></listitem>
   </varlistentry>
   <varlistentry>
+    <term><varname>types.path</varname></term>
+    <listitem><para>A filesystem path, defined as anything that when coerced to 
+        a string starts with a slash. Even if derivations can be considered as 
+        path, the more specific <literal>types.package</literal> should be 
+        preferred.</para></listitem>
+  </varlistentry>
+  <varlistentry>
+    <term><varname>types.package</varname></term>
+    <listitem><para>A derivation or a store path.</para></listitem>
+  </varlistentry>
+</variablelist>
+
+<para>Integer related types:</para>
+
+<variablelist>
+  <varlistentry>
+    <term>
+      <varname>types.ints.signed</varname>
+    </term>
+    <listitem><para>A signed integer.
+    </para></listitem>
+  </varlistentry>
+  <varlistentry>
+    <term>
+      <varname>types.ints{signed8, signed16, signed32}</varname>
+    </term>
+    <listitem>
+      <para>Signed integers with a fixed length (8, 16 or 32 bits).
+        They go from
+        <inlineequation><mathphrase>−2<superscript>n</superscript>/2</mathphrase>
+        </inlineequation> to <inlineequation>
+        <mathphrase>2<superscript>n</superscript>/2−1</mathphrase>
+        </inlineequation>
+        respectively (e.g. <literal>−128</literal> to <literal>127</literal>
+        for 8 bits).
+    </para></listitem>
+  </varlistentry>
+  <varlistentry>
     <term><varname>types.int</varname></term>
-    <listitem><para>An integer.</para></listitem>
+    <listitem><para>A convenience alias for <literal>ints.signed</literal>.
+    </para></listitem>
   </varlistentry>
   <varlistentry>
     <term>
-      <varname>types.intBetween</varname>
-      <replaceable>min</replaceable>
-      <replaceable>max</replaceable>
+      <varname>types.ints.unsigned</varname>
     </term>
-    <listitem><para>An integer between <replaceable>min</replaceable>
-        and <replaceable>max</replaceable> (both inclusive).
-        Useful for e.g. port ranges.
+    <listitem><para>An unsigned integer (that is ≥ 0).
```

Comment:
Hmm... I'm not sure we should put unicode here, given these docs are sometimes read on TTY8.

Reply:
Doesn’t seem to be a problem for these:
![screenshot](https://user-images.githubusercontent.com/3153638/32191363-1bb660ac-bdb1-11e7-9ca6-4b5a8e682388.png)

Do you refer to the installation-manual tty? Is that not in UTF-8?

Reply:
Removed it, since it’s mandatory that it shows up everywhere.